### PR TITLE
Feature/cor 1357 rename verpleeghuiszorg to new name

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
**Before:**
<details>
  <summary>Toggle before screenshot</summary>

Screenshot 2023-01-24 at 12 37 36

</details>

**After:**
<details>
  <summary>Toggle after screenshot</summary>

Screenshot 2023-01-24 at 12 37 49

Screenshot 2023-01-24 at 12 37 56

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
For the sake of getting this on develop ASAP, I did not pay attention to anything magic number's related in this PR.

File: packages/app/schema/nl/__difference.json [hunk 1]
```diff
@@ -39,13 +39,19 @@
     "sewer__average": {
       "$ref": "#/definitions/diff_integer"
     },
-    "nursing_home__newly_infected_people": {
+    "vulnerable_nursing_home__infected_locations_total": {
```
Comment (VWSCoronaDashboard21) [inline packages/app/schema/nl/__difference.json:42]: Should this also have the archived naming?

File: packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx [hunk 2]
```diff
@@ -101,79 +102,121 @@ function VulnerableGroups(props: StaticProps<typeof getStaticProps>) {
...
-              dateOrRange: nursinghomeDataLastValue.date_unix,
+              dateOrRange: vulnerableNursingHomeDataLastValue.date_unix,
               dateOfInsertionUnix: lastInsertionDateOfPage,
               dataSources: [positiveTestedPeopleText.bronnen.rivm],
             }}
             referenceLink={positiveTestedPeopleText.reference.href}
             articles={content.articles}
           />
 
+          {textShared.osiris_archiving_notification && <Markdown content={`> > ${textShared.osiris_archiving_notification}`} />}
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx:117]: What is the `> > ${...}` for? 🤔

File: packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx [hunk 3]
```diff
@@ -278,78 +321,82 @@ function VulnerableGroups(props: StaticProps<typeof getStaticProps>) {
             />
           </ChartTile>
 
-          <Divider />
+          {false && (
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx:324]: I think this change is not intentional and should be either removed or reverted?

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Pre-approved, but do have a look at some minor comments.

File: packages/app/schema/vr/nursing_home_archived_20230126.json [hunk 4]
```diff
@@ -17,7 +17,7 @@
   "additionalProperties": false,
   "definitions": {
     "value": {
-      "title": "vr_nursing_home_value",
+      "title": "vr_nursing_home_archived_20230126_value",
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/schema/vr/nursing_home_archived_20230126.json:20]: picked up in cor-1361

File: packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx:117]: picked/explained up in cor-1361

File: packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx (hunk 3 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/kwetsbare-groepen-70-plussers.tsx:324]: picked/explained up in cor-1361

File: packages/app/src/next-config/redirects/redirects.js [hunk 5]
```diff
@@ -76,17 +75,26 @@ async function redirects() {
...
       permanent: false,
     },
     {
-      source:
-        '/actueel/veiligheidsregio/:vr(vr|VR|vR|Vr):nr(\\d{2}):slash(/{0,1}):page*',
+      source: '/actueel/veiligheidsregio/:vr(vr|VR|vR|Vr):nr(\\d{2}):slash(/{0,1}):page*',
       destination: '/veiligheidsregio/VR:nr',
       permanent: false,
     },
+    // Redirects for OSIRS change
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/next-config/redirects/redirects.js:87]: picked/explained up in cor-1361

File: packages/common/src/types/data.ts [hunk 6]
```diff
@@ -586,7 +591,29 @@ export interface NlNursingHomeValue {
...
+  date_of_insertion_unix: number;
+}
+export interface NlVulnerableHospitalAdmissions {
+  admissions_age_70_plus: number;
+  date_start_unix: number;
+  date_end_unix: number;
+  date_of_insertion_unix: number;
+}
+export interface NlDisabilityCareArchived_20230126 {
   values: NlDisabilityCareValue[];
```
Comment (VWSCoronaDashboard21) [inline packages/common/src/types/data.ts:617]: Should this have the archiving naming as well?

File: packages/common/src/types/data.ts [hunk 7]
```diff
@@ -769,7 +796,7 @@ export interface NlDeceasedCbsValue {
   date_end_unix: number;
   date_of_insertion_unix: number;
 }
-export interface NlElderlyAtHome {
+export interface NlElderlyAtHomeArchived_20230126 {
   values: NlElderlyAtHomeValue[];
```
Comment (VWSCoronaDashboard21) [inline packages/common/src/types/data.ts:800]: Should this have the archiving naming as well?

File: packages/common/src/types/data.ts [hunk 8]
```diff
@@ -571,7 +576,7 @@ export interface NlTestedGgdArchivedValue {
   date_unix: number;
   date_of_insertion_unix: number;
 }
-export interface NlNursingHome {
+export interface NlNursingHomeArchived_20230126 {
   values: NlNursingHomeValue[];
```
Comment (VWSCoronaDashboard21) [inline packages/common/src/types/data.ts:580]: Should this have the archiving naming as well?

File: packages/common/src/types/data.ts [hunk 9]
```diff
@@ -586,7 +591,29 @@ export interface NlNursingHomeValue {
   date_unix: number;
   date_of_insertion_unix: number;
 }
-export interface NlDisabilityCare {
+export interface NlVulnerableNursingHome {
+  values: NlVulnerableNursingHomeValue[];
```
Comment (VWSCoronaDashboard21) [inline packages/common/src/types/data.ts:595]: Should this have the archiving naming as well?

File: packages/common/src/types/data.ts [hunk 10]
```diff
@@ -295,14 +298,16 @@ export interface NlDifference {
   doctor__covid_symptoms_per_100k: DifferenceDecimal;
   doctor__covid_symptoms: DifferenceInteger;
   sewer__average: DifferenceInteger;
-  nursing_home__newly_infected_people: DifferenceInteger;
-  nursing_home__infected_locations_total: DifferenceInteger;
-  nursing_home__deceased_daily: DifferenceInteger;
+  vulnerable_nursing_home__infected_locations_total: DifferenceInteger;
+  nursing_home__newly_infected_people_archived_20230126: DifferenceInteger;
+  nursing_home__deceased_daily_archived_20230126: DifferenceInteger;
+  vulnerable_tested_per_age_group: DifferenceInteger;
```
Comment (VWSCoronaDashboard21) [inline packages/common/src/types/data.ts:304]: Nitpicky and not blocking but I think the format can be made consistent by using:
vulnerable__tested_per_age_group:
vulnerable__hospital_admissions:

File: packages/app/src/pages/veiligheidsregio/[code]/kwetsbare-groepen-70-plussers.tsx [hunk 11]
```diff
@@ -39,7 +32,13 @@ export { getStaticPaths } from '~/static-paths/vr';
 export const getStaticProps = createGetStaticProps(
   ({ locale }: { locale: keyof Languages }) => getLokalizeTexts(selectLokalizeTexts, locale),
   getLastGeneratedDate,
-  selectVrData('difference.nursing_home__deceased_daily', 'difference.nursing_home__infected_locations_total', 'difference.nursing_home__newly_infected_people', 'nursing_home'),
+  selectVrData(
+    'difference.nursing_home__deceased_daily_archived_20230126',
+    'difference.vulnerable_nursing_home__infected_locations_total',
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/pages/veiligheidsregio/[code]/kwetsbare-groepen-70-plussers.tsx:37]: Should this contain the archived naming as well?

**Review by VWSCoronaDashboard21 — APPROVED** [review]

**Pull request merged**